commit 9344a5b2a8ed7b539d62cce595431b34e28ffcf5 (refs/changes/61/661/2)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2016-10-13T01:56:00+00:00 (3 years ago)
    
    DOCKER-949 Adjust set of characters escaped for NAPI
    Reviewed by: Josh Wilsdon <jwilsdon@joyent.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/lib/backends/sdc/utils.js b/lib/backends/sdc/utils.js
index 4feaa12..7f34502 100644
--- a/lib/backends/sdc/utils.js
+++ b/lib/backends/sdc/utils.js
@@ -982,7 +982,7 @@ function compressPorts(ports) {
 // XXX - workaround for NAPI-367
 function ldapEscape(str) {
     /* JSSTYLED */
-    return str.replace(/[\-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, '\\$&');
+    return str.replace(/[=\(\)\*\\]/g, '\\$&');
 }
 
 // ---- exports
